docs/ManuallyRunning.md

```diff
@@ -0,0 +1,237 @@
+
+```shell script
+$ export SELECTED_TESTS_MONGO_URI=localhost:27017
+$ init-mongo create-indexes
+{"message": "Adding indexes for collection", "lineno": 24, "filename": "datasource_cli.py", "collection": "test_mappings_queue", "logger": "selectedtests.datasource.datasource_cli", "level": "info"}
+{"message": "Adding indexes for collection", "lineno": 24, "filename": "datasource_cli.py", "collection": "task_mappings_queue", "logger": "selectedtests.datasource.datasource_cli", "level": "info"}
+{"message": "Adding indexes for collection", "lineno": 45, "filename": "datasource_cli.py", "collection": "test_mappings", "logger": "selectedtests.datasource.datasource_cli", "level": "info"}
+{"message": "Adding indexes for collection", "lineno": 45, "filename": "datasource_cli.py", "collection": "task_mappings", "logger": "selectedtests.datasource.datasource_cli", "level": "info"}
+{"message": "Adding indexes for collection", "lineno": 73, "filename": "datasource_cli.py", "collection": "test_mappings_test_files", "logger": "selectedtests.datasource.datasource_cli", "level": "info"}
+{"message": "Adding indexes for collection", "lineno": 60, "filename": "datasource_cli.py", "collection": "task_mappings_tasks", "logger": "selectedtests.datasource.datasource_cli", "level": "info"}
+```
+
+**Note** from here on, we will assume that **SELECTED_TESTS_MONGO_URI** is set.
+
+## Launch Web Service
+
+```shell script
+$ export EVG_API_USER=$(python -c "import yaml; import sys; cfg = yaml.safe_load(sys.stdin); print(cfg['user'])" < ~/.evergreen.yml)
+$ export EVG_API_KEY=$(python -c "import yaml; import sys; cfg = yaml.safe_load(sys.stdin); print(cfg['api_key'])" < ~/.evergreen.yml)
+$ export SELECTED_TESTS_MONGO_URI="localhost:27017"
+$ uvicorn --host 0.0.0.0 --port 8080 --workers 1 selectedtests.app.asgi:app --reload
+```
+
+__Note__: reload is only used in development mode.
+
+## Generate test and task mappings 
+
+Use the following commands to create the test and task mappings for **mongodb-mongo-master**.
+
+```shell script
+$ export SELECTED_TESTS_URI=http://localhost:8080
+$ export SELECTED_TESTS_PROJECT=mongodb-mongo-master 
+$ curl  -X POST ${SELECTED_TESTS_URI}/projects/${SELECTED_TESTS_PROJECT}/test-mappings \
+        -d '{"source_file_regex":"^src/mongo", "test_file_regex":"^jstests"}' \
+        -H "Content-Type: application/json"
+$ curl -X POST ${SELECTED_TESTS_URI}/projects/${SELECTED_TESTS_PROJECT}/task-mappings \
+       -H 'accept: application/json' \
+       -H 'Content-Type: application/json' \
+       -d '{ "source_file_regex": "^src/mongo", "module": "enterprise", "module_source_file_regex": "^src", "build_variant_regex": "^!"}'
+``` 
+
+Now that the mappings have been created, we must populate them.
+
+## Populating the mappings.
+ 
+You should run the `process-test-mappings` and `process-task-mappings` commands once every
+day (see [Generate test and task mappings](#generate-test-task-mappings)) . This will gather the
```

Comment:
`process-test-mappings` and `process-task-mappings` only need to be run once, not every day